tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that systemd timer is used for time synchronization.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+    record_info("Check yast-timesync.timer", "Check if file /etc/systemd/system/yast-timesync.timer exists and has expected configuration.");
+    assert_script_run('ls /etc/systemd/system/yast-timesync.timer');
+    my $conf_file = script_output("cat /etc/systemd/system/yast-timesync.timer");
+    $conf_file =~ /OnUnitActiveSec=(?<interval>(\S+))min/;
+    assert_equals($test_data->{profile}->{'ntp-client'}->{ntp_sync}, $+{interval}, "The interval in yast-timesync configuration file is not the expected one.");
+    sleep 61;
+    record_info("Check logs for timesync", "Check if the use of systemd timers can be spotted in the journal logs as it's expected.");
+    assert_script_run('journalctl -u yast-timesync --since "2 minutes ago" | grep "Started One time sync configured by YaST"');
+    record_info("Check logs for time deviation fix", "Check if \"One time synchronization\" will fix time deviation as expected.");
+    assert_script_run('date $(date +%m%d%H%M%Y.%S -d "1 minute ago")');
```

Comment:
I could not see visible in the output the current date before the operation, it would be nice to debug to see both.

Reply:
ok